demos/dqx_demo_library.py

```diff
@@ -444,6 +444,142 @@
 
 # COMMAND ----------
 
+# MAGIC %md
+# MAGIC ## Applying quality checks and writing to Delta tables
+
+# COMMAND ----------
+
+from databricks.labs.dqx import check_funcs
+from databricks.labs.dqx.engine import DQEngine
+from databricks.labs.dqx.rule import DQRowRule
+from databricks.sdk import WorkspaceClient
+
+# Create sample data
```

Comment:
I would include it in the existing examples instead of creating completely new. The demo starts to be quite long. We don't need to repeat. You can add it to the section "Applying quality checks in the Lakehouse medallion architecture", as alternative approach to do both quality checking and saving.